lib/sidekiq/form526_job_status_tracker/job_tracker.rb

```diff
@@ -91,7 +91,8 @@ def with_tracking(job_title, saved_claim_id, submission_id, is_bdd = nil)
           exception_raised = true
           raise
         ensure
-          job_success unless exception_raised
+          job_status = Form526JobStatus.find_by(job_id: jid)
+          job_success unless exception_raised || job_status&.error_class
```

Comment:
This additional evaluation was needed to address adjustments made to `with_tracking` in `SubmitForm526#perform`.